Makefile

```diff
@@ -25,6 +25,12 @@ proto:
 	for dir in $$(git ls-files '*.proto' | xargs -n1 dirname | uniq); do \
 		protoc -I $$dir --go_out=plugins=grpc:$$dir $$dir/*.proto; \
 	done
+	cat grpclb/grpc_lb_v1/load_balancer.pb.go | \
```

Comment:
The server side generated code is for testing only, and client side code was moved to `grpc` package to avoid the dependency problem. They can be considered hand-written. So for `grpclb.proto`, we could just generate the messages, not the services (call protoc without the grpc plugin).

And also, it may be a good idea to use `go generate` to generate this `pb.go` files. We can add the `//go:generate` comment to those go files that import `pb` packages.